components/codeblock/codeblock.vue

```diff
@@ -0,0 +1,20 @@
+<template>
+  <pre>
+    <code
+      class="d-d-block d-bgc-white d-ws-pre-wrap d-p8 d-my4 d-ba d-bc-black-500 d-bar8 d-fs-100 d-lh-200 d-fc-red-400"
```

Comment:
A couple styling suggestions
- Set the padding to only `d-p8` remove `d-my4`
- Set the font size to `d-fs-200`
- Set the line-height to `d-lh-400`
- I understand we will be looking at using a library for syntax highlighting but the fall back font color should = `d-fc-secondary`
- Border color = `border - subtle` (not sure what the new border token class are)
- Background = `d-bgc-secondary`

Reply:
Love the suggestions 👏 thanks Brad!
<img width="1512" alt="image" src="https://user-images.githubusercontent.com/65984290/227631184-1bb5c814-b8a2-4bd3-943e-21595f49d5f9.png">